tlv/fuzz_test.go

```diff
@@ -96,3 +96,42 @@ func FuzzVarBytes(f *testing.F) {
 		harness(t, data, EVarBytes, DVarBytes, &val, uint64(len(data)))
 	})
 }
+
+// bigSizeHarness works the same as harness, except that it compares decoded
+// values instead of encoded values. We do this because DBigSize may leave some
+// bytes unparsed from data, causing the encoded data to be shorter than the
+// original.
+func bigSizeHarness(t *testing.T, data []byte, val1, val2 interface{}) {
+	if len(data) > 9 {
```

Comment:
I'm starting to wonder how often this is hit and whether it deteriorates the efficacy of our test?